docs/installation/install-livepeer/installing-for-development.md

```diff
@@ -33,7 +33,7 @@ Building `livepeer` requires some system dependencies.
 Linux (Ubuntu: 16.04 or 18.04):
 
 ```bash
-apt-get update && apt-get -y install build-essential pkg-config autoconf git curl
+apt-get update && apt-get -y install build-essential pkg-config autoconf git curl protobuf-compiler-grpc golang-goprotobuf-dev
```

Comment:
I'm guessing that this isn't necessary on v16 and v18 of Ubuntu - could you keep the original command in place and specify that the update is necessary to make it work for v20?